### DISCUSSION TITLE
I can't hear someone / someone can't hear me

### DISCUSSION THREAD

**Discussion by inozemtsev-roman** [tonmetaspace/.github#31] [Q&A]
A connectivity issue may result in one person in the room being unable to hear one or more other participants in a space. We first recommend that you confirm that you can hear other sounds in the room (for example, the sound an object makes when you add it to the room, a video in the room, or the chat sound) and that your Metaspace audio preferences aren't set to 0 volume. Also confirm that you are not muted and that you have allowed microphone permissions for Metaspace. If you aren't getting any audio at all, try reconnecting to the room with a different audio output device. If you are willing, we also ask that you fill out this form with network connectivity logs, which will help us diagnose system issues related to audio and connectivity.

## Please fill out this form if you are experiencing issues with WebRTC (voice audio, webcam, and desktop streaming) in Metaspace.
1. What operating system are you using?
* Windows 10
* macOS
* Linux
* iOS
* Android
* Oculus Quest
2. What browser are you using?
* Chrome
* Safari
* Firefox
* Oculus Browser
3. Please describe what was happening when the issue(s) occurred, and what the issue(s) you experienced were
4.  If you are on Firefox or Chrome for desktop, please upload your webrtc log. In Firefox: open a new tab and navigate to about:webrtc and click on "Save Page" in the top left corner. In Chrome: open a new tab and navigate to chrome://webrtc-internals and click on "Create Dump" then "Download the PeerConnection updates and stats data".

![image](https://user-images.githubusercontent.com/24755187/224569797-667f4cc1-dd2b-4f9d-8434-db16be7c403e.png)

![image](https://user-images.githubusercontent.com/24755187/224569807-612af5a0-6d7d-4ca4-9809-8154f0bdde83.png)

**Comment by kerrymatten** [comment]
no problem here
